app/scripts/templates/settings/change_password.mustache

```diff
@@ -18,7 +18,7 @@
         <label class="label-helper"></label>
         <input type="password" class="password" id="old_password" placeholder="{{#t}}Old password{{/t}}" required pattern=".{8,}" autofocus />
 
-        <input id="show-old-password" type="checkbox" class="show-password" aria-controls="old_password" tabindex="-1" data-novalue />
+        <input id="show-old-password" type="checkbox" class="show-password" aria-controls="old_password" tabindex="-1" data-novalue data-unsynchronized-input="true"/>
```

Comment:
I wish there were an emoji for [bike-shed](https://en.wikipedia.org/wiki/Law_of_triviality) because that's what I'm about to do. I wonder if reversing the semantics would be clearer. Instead of `data-unsynchronized-show`, maybe `data-synchronize-show` makes the intent explicit and synchronization between fields is opt-in. In that scenario, change-password would not need to change, reset-password would.